### PR TITLE
ID-199 Fix blurry masthead image in Safari

### PR DESCRIPTION
**Pull request by acursley**
Adds @2x and @3x PNGs, and serves them to Safari in preference to the SVG version, which appears blurry when used as a background image.

### CODE REVIEW

File: images/generate-assets.sh [hunk 1]
```diff
@@ -1,7 +1,5 @@
 #!/bin/bash
 
-# These assets are used almost exclusively for IE8 fallback, so don't need retina versions
```
Comment (matmannion) [inline images/generate-assets.sh]: Is there another change to this file I'm missing to generate the @2x and @3x versions?

File: Gruntfile.js [hunk 2]
```diff
@@ -89,7 +89,8 @@ module.exports = function (grunt) {
           'js/not-selector-feature-detect.modernizr.js',
           'js/style-update-feature-detect.modernizr.js',
           'js/svg-filters-feature-detect.modernizr.js',
-          'js/ie10-viewport-bug-workaround.js'
+          'js/ie10-viewport-bug-workaround.js',
+          'js/safari-user-agent-test.js'
```
Comment (matmannion) [inline Gruntfile.js]: Can we have `.modernizr.` in the filename and re-order it before the viewport bug workaround so it goes with the other modernizr tests?

**Comment by matmannion** [comment]
LGTM :shipit:

File: images/generate-assets.sh (hunk 1 above)
Comment (acursley) [inline images/generate-assets.sh]: Hidden in a commit message: `SCALE=2 ./generate-assets.sh`, `SCALE=3 ./generate-assets.sh`.

**Pull request merged**